src/foam/nanos/export/ExportDriverRegistry.js

```diff
@@ -42,6 +42,10 @@ foam.CLASS({
       class: 'Boolean',
       name: 'isOpenable'
     },
+    {
+      class: 'Boolean',
+      name: 'isAllColumnOptionEnabled',
```

Comment:
Can you add documentation to this. please.

Reply:
fixed